modules/init-snippet-attach-ebs-volume/snippet.tpl

```diff
@@ -30,5 +22,3 @@ else
   dev_name="/dev/$(readlink "$${dev_id}" | tr / '\n' | tail -1)"
   [ "$${dev_name}" == "${device_path}" ] || ln -s "$${dev_name}" "${device_path}"
 fi
-
-${init_suffix}
```

Comment:
Why is this dropped?

Reply:
This part (and init_prefix) is moved to the output part. So this snippet is focused on attaching EBS.